### ISSUE TITLE
GLFW throws error when start gles-worker desktop on Mac

### ISSUE THREAD

**Issue by jiradeto** [google/graphicsfuzz#390]
**Environment**

- os.name: Mac OS X
- os.version: 10.13.4
- GL_VERSION : 2.1 INTEL-10.32.48
- GL_SHADING_LANGUAGE_VERSION: 1.20
- GL_VENDOR: Intel Inc
- GL_RENDERER: Intel Iris OpenGL Engine
- Java version: 1.8.0_191
- Python version: 3.7.2

**Description**

I built the gles-worker and tried to execute with command `java -ea -jar gles-worker-desktop.jar`. However, the GLFW library failed to create window and throw an error.

The error log: 
```Java
java.lang.ExceptionInInitializerError
	at org.lwjgl.glfw.GLFW.glfwCreateWindow(GLFW.java:1647)
	at com.badlogic.gdx.backends.lwjgl3.Lwjgl3Application.createGlfwWindow(Lwjgl3Application.java:442)
	at com.badlogic.gdx.backends.lwjgl3.Lwjgl3Application.createWindow(Lwjgl3Application.java:391)
	at com.badlogic.gdx.backends.lwjgl3.Lwjgl3Application.createWindow(Lwjgl3Application.java:379)
	at com.badlogic.gdx.backends.lwjgl3.Lwjgl3Application.<init>(Lwjgl3Application.java:108)
	at com.graphicsfuzz.glesworker.desktop.DesktopLauncher.start(DesktopLauncher.java:263)
	at com.graphicsfuzz.glesworker.desktop.DesktopLauncher.main(DesktopLauncher.java:164)
Caused by: java.lang.IllegalStateException: GLFW windows may only be created on the main thread and that thread must be the first thread in the process. Please run the JVM with -XstartOnFirstThread. For offscreen rendering, make sure another window toolkit (e.g. AWT or JavaFX) is initialized before GLFW.
	at org.lwjgl.glfw.EventLoop$OffScreen.<clinit>(EventLoop.java:37)
	... 7 more

```

**Current Workaround**

- For running via command line, use `java -XstartOnFirstThread -ea -jar gles-worker-desktop.jar`
- For running on IntelliJ IDEA, set `-XstartOnFirstThread` as the VM options for the configuration.

**Comment by afd** [comment]
This might have been fixed in a newer version of LibGDX.  Could you test that out please?

**Comment by paulthomson** [comment]
@jiradeto Does the workaround actually work for you? On desktop, I believe we invoke the `java` binary again as a child process without forwarding the JVM args. We should change the invocation to include -XstartOnFirstThread when running on MacOS (and to include -ea always!). You can see this here:

https://github.com/google/graphicsfuzz/blob/master/gles-worker/desktop/src/com/graphicsfuzz/glesworker/desktop/DesktopLauncher.java#L171

We should split it into:

* cmd.addAll "java" and JVM args "-ea" and "-XX:ErrorFile=...".
* if on macOS: cmd.addAll "-XstartOnFirstThread".
* cmd.addAll "-jar" and jarPath
* cmd.addAll args (existing args input)

Could you:

1. See if the workaround you provided works. If yes, then that would be surprising and we should investigate it further before doing step 2.
2. Try doing the above change and see if that works for you. If yes, then do a PR and I can check if it works for me as well.

**Comment by afd** [comment]
@paulthomson does #469 fix this?

**Comment by paulthomson** [comment]
Closed by #469
